server/text_generation_server/layers/attention/rocm.py

```diff
@@ -112,24 +139,44 @@ def paged_attention(
         )
         max_logits = torch.empty_like(exp_sums)
 
-        ops.paged_attention_v2(
-            out,
-            exp_sums,
-            max_logits,
-            tmp_output,
-            query,
-            key_cache,
-            value_cache,
-            kv_head_mapping,
-            softmax_scale,
-            block_tables,
-            input_lengths,
-            block_size,
-            max_s,
-            None,
-            "auto",
-            1.0,
-        )
+        if not use_custom:
+            ops.paged_attention_v2(
```

Comment:
What about softcapping (in both custom/not custom cases)? If it is not supported, we should raise an exception somewhere when someone tries to use a model with softcapping.

Reply:
Its not supported, added an error in case provided